index.js

```diff
@@ -69,7 +69,7 @@ function init(opt) {
 
     // overwrite default options
 
-    migrationDir = path.resolve(path.dirname(require.main.filename), (opt.migrationDir || migrationDir));
+    migrationDir = path.resolve(opt.migrationDir || migrationDir);
```

Comment:
Let's stick to the `require.main.filename` thing because this way the path to the migration dir can be specified (and is resolved) from the main module (e.g. `xxx_server.js`) of the app using this lib and does not depend purely on the working directory - although you could solve this in the configuration of the `migrationDir` itself with a construct like 

    migrationDir: path.resolve(__dirname, 'migrations')

However, I prefer "my" variant ;)